concepts/Common/Licensing/25 Validation.md

```diff
@@ -0,0 +1,40 @@
+[note] DevExtreme’s license verification process does not impact app functionality nor affect app performance. 
+
+When creating the first component, DevExtreme checks the license key specified in the `config()` method. If license key verification is successful, you will not see any additional license-related messages within the browser console.
+
+If license key validation fails, DevExtreme will display one of the following messages within the browser console.
+
+<table class="dx-table full-width">
+    <tr>
+        <th>Case</th>
+        <th>Message</th>
+    </tr>
+    <tr>
+        <td>Valid license key not found.</td>
+        <td>
+        <code>W0019 - DevExtreme: Unable to Locate a Valid License Key</code><br><br>
```

Comment:
This page also has to have the warnings https://js.devexpress.com/React/Documentation/23_1/ApiReference/Common/Utils/Errors_and_Warnings/

Reply:
Warnings are generated by developers, need to ask @IlyaKhD